src/api/posts/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@senecacdot/satellite": "^1.x",
+    "dotenv": "^10.0.0",
```

Comment:
This change doesn't seem necessary

Reply:
Also need to revert this.

Reply:
`git checkout master src/api/posts/package.json`